libopae/pluginmgr.c

```diff
@@ -135,12 +135,38 @@ STATIC char *find_cfg()
 	return NULL;
 }
 
+STATIC void *opae_plugin_mgr_find_plugin(const char *lib_path)
+{
+	char plugin_path[PATH_MAX];
+	const char *search_paths[] = {
+		"/usr/lib64/opae/",
+		"/usr/lib/opae/",
+		""
+	};
+	unsigned i;
+	void *dl_handle;
+
+	for (i = 0 ;
+		i < sizeof(search_paths) / sizeof(search_paths[0]) ; ++i) {
```

Comment:
I think I may have seen newer compilers warn about this...

Reply:
It builds cleanly in Fedora with g++ 8.3.1.